pkg/test_samples/auth_secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+data:
+  .dockerconfigjson: eyJhdXRocyI6eyJ0ZXN0LnJlZ2lzdHJ5LmlvIjp7InVzZXJuYW1lIjoidGVzdF91c2VyIiwicGFzc3dvcmQiOiJ0ZXN0X3Bhc3MiLCJhdXRoIjoiZEdWemRGOTFjMlZ5T25SbGMzUmZjR0Z6Y3c9PSJ9fX0=
+kind: Secret
+metadata:
+  name: oci-registry-cred
+  namespace: default
+  labels:
+    operator.kyma-project.io/oci-registry-cred: test-operator
```

Comment:
```suggestion
    operator.kyma-project.io/dockerconfigjson-for: test-operator
```

Reply:
as agreed, we not limit the naming of the labels, so here is just an example, user can freely choose what label they want.